Lib/types.py

```diff
@@ -143,6 +143,35 @@ def _calculate_meta(meta, bases):
                         "of the metaclasses of all its bases")
     return winner
 
+
+def get_original_bases(cls, /) -> tuple[type, ...] | None:
```

Comment:
This type annotation isn't accurate. If a class inherits from `list[int]`, you'll have a `GenericAlias` object in the tuple of objects that's returned. And if a class inherits from `typing.NamedTuple`, you'll get a _function_ object in the tuple of types that's returned:

```pycon
>>> from typing import *
>>> class Foo(NamedTuple):
...     x: int
...
>>> Foo.__orig_bases__
(<function NamedTuple at 0x000002C3DE0432E0>,)
```

We'll probably end up annotating this function as `-> tuple[Any, ...] | None` in typeshed. Most of the functions in this module don't have type annotations -- maybe we should just omit the return annotation here, for simplicity?

```suggestion
def get_original_bases(cls, /):
```